brave-intl/publishers
# Patch for Partner Stats 🏷

## Patch for Partner Stats

![Screen Shot 2019-04-18 at 4 33 29 PM](https://user-images.githubusercontent.com/4713771/56389520-c3204880-61f7-11e9-97b1-34f63219f924.png)


#### Features

* Fix an issue partners were seeing with the stats on their dashboard. 
* Aggregate all stats for a referral code, instead of showing daily stats
* Add additional error handling for null or empty stats
* Ensure reporting works for partner generated campaigns/codes 

#### How To Test

1. As a `partner` create a campaign with promo codes 
2. As an `admin`, go to the referral promo page in the dashboard
3. Generate a report to update the stats
4. Ensure parity between partner stats, admin stats, and reports


#### Pull Request Checklist
